cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/query/IndexManager.java

```diff
@@ -255,7 +255,7 @@ public String ensureIndexed(List<Object> fieldNames, String indexName) {
      *  @param indexType The type of index (json or text currently supported)
      *  @return name of created index
      */
-    public String ensureIndexed(List<Object> fieldNames, String indexName, String indexType) {
```

Comment:
Since we are prior to 1.0 I thought it made sense to make the change breaking, and not carry the legacy through, otherwise we'd be stuck with it until 2.0, and based on the level of confusion caused I think its a safer bet to remove & replace for 1.0.